### PR TITLE
US138621 - Add OneDrive LTI file picker button

### PR DESCRIPTION
**Pull request by DerekP-d2l**
[US138621](https://rally1.rallydev.com/#/520974201756d/iterationstatus?detail=%2Fuserstory%2F633260836465&fdp=true&view=3265e82f-7ab0-4a61-abd2-b40776bb4cb3)
Adding an additional OneDrive button, for a new LTI file picker. This will eventually replace the existing OneDrive picker completely, but for now both are needed.

Related LMS changes are [here](https://github.com/Brightspace/lms/pull/23490).

### CODE REVIEW

**Review by JeremyRumphD2L — APPROVED** [review]

File: demo/d2l-activity-editor/d2l-activity-assignment-editor/data/attachmentsCollection.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     {
       "type": "application/x-www-form-urlencoded",
       "href": "https://8b4b0787-25ca-43d2-89ab-b9495546ad09.assignments.api.proddev.d2l/6613/folders/3/attach-link",
-      "name": "add-onedrive-link",
+      "name": "add-onedrivelti-link",
```
Comment (amauryperez) [inline demo/d2l-activity-editor/d2l-activity-assignment-editor/data/attachmentsCollection.json:75]: any particular reason why we are replacing onedrive with onedrivelti?

Comment (amauryperez) [reply demo/d2l-activity-editor/d2l-activity-assignment-editor/data/attachmentsCollection.json:75]: oh, it is a demo page, so I guess we dont care about onedrive only anymore

Comment (DerekP-d2l) [reply demo/d2l-activity-editor/d2l-activity-assignment-editor/data/attachmentsCollection.json:75]: Yeah exactly, I thought it'd make sense to use the newer version in the demo page.

**Review by amauryperez — APPROVED** [review]
LGTM. Can you make sure this is tested in the context of Consistent Eval?

**Pull request merged**